/**:
  ros__parameters:
    use_sim_time: false
    max_steer_angle: 470.0
    publish_my_laps: true
    vehicle_number: 7